[Magic WAN] High availability configurations (#13913)
cloudflare/cloudflare-docs: content/magic-wan/configuration/connector/configure-hardware-connector.md

2. Unplug the physical connection to the Internet-connected device which provides DHCP.
3. Adjust your physical connections as required to match the configuration specified in the [site configuration](#1-create-a-site) step (for example, static IP WAN plugged into a physical port with no DHCP connection).
4. Power cycle the Connector.

---

## About high availability configurations

When you set up a site in high availability, the WANs and LANs in your Connectors have the same configuration but are replicated on two nodes. In case of failure of a Connector, the other Connector becomes the active node, taking over configuration of the LAN gateway IP and allowing traffic to continue without disruption.

Because Connectors in high availability configurations share a single site, you need to set up:

- **Static address**: The prefix for the primary node in your site.
- **Secondary static address**: The prefix for the secondary node in your site.
- **Virtual static address**: The prefix that the LAN south of the Connector will forward traffic to, which is the LANs gateway IP.

Make sure all prefixes are part of the same subnet.

### ​​Create a high availability configuration

> You cannot enable high availability for an existing site. If you decide to enable high availability for an existing site in the Cloudflare dashboard, you need to delete it and start again.

Reviewer: ```suggestion
You cannot enable high availability for an existing site. To add high availability to an existing site in the Cloudflare dashboard, you need to delete the site and start again.
```

To set up a high availability configuration:

1. Follow the steps in [Create a site](#1-create-a-site) up until step 4.
2. After naming your site, select **Enable high availability**.
3. Under **Connector**, select **Add Connector**.
4. From the list, choose your first Connector > **Add Connector**.
5. Back on the previous screen, select **Add secondary Connector**.
6. From the list, choose your second Connector > **Add Connector**.
7. Select **Next** to [Create a WAN](#2-create-a-wan).
8. To create a LAN, follow the steps mentioned above in [Create a LAN](#3-create-a-lan) up until step 4.

> 9. In **Static address**, enter the prefix for the primary node in your site. For example `192.168.10.1/24`.

Reviewer: ```suggestion
9. In **Static address**, enter the prefix for the primary node in your site. For example, `192.168.10.1/24`.
```
